.env

```diff
@@ -0,0 +1 @@
+GITHUB_PERSONAL_ACCESS_TOKEN = 9588bccba362f2e27deb0e1f16ccabe97eefdcd4
```

Comment:
Remove this file. We can instead add a section in the README instructing users to create a `.env` file and inserting a `GITHUB_PERSONAL_ACCESS_TOKEN`. For inspiration, we write something similar in https://github.com/taskcluster/taskcluster-web/blob/master/README.md#environment.